Automattic/wp-calypso
# Marketing -> Traffic settings: Fix SEO & Analytics Settings for Free plans

#### Changes proposed in this Pull Request

Show the correct banners on Tools > Marketing > Traffic > Search Engine Optimization and Google Analytics:
- Use `hasActiveSiteFeature()` to check if we should show a banner / nudge or the setting
- Use `isAtomic` to show the correct nudge or form

#### Testing instructions



* On a free Atomic site, open Tools > Marketing > Traffic settings.
* There should be upsell nudges under Search engine optimization and Google Analytics.
* Clicking the upsell nudges goes to plans, not checkout.
* Repeat for Personal and Premium plans on Atomic sites. Google Analytics should be available with no nudge on Premium Atomic.
* Regression: 
    * Regular/paid Atomic: SEO and Analytics should be available, no nudge.
    * Simple: The nudges should be the same as for free, Personal, and Premium Atomic.
    * VIP (see: PCYsg-zan-p2): SEO has a nudge with no plan, Google Analytics is available.
    * Jetpack: SEO should be available on all plans, Analytics is not available on free Jetpack and there should be a Jetpack checkout nudge.

Simple | Free Atomic | Regular Atomic
------- | ------------ | ---------------
<img width="1056" alt="Screen Shot 2021-06-01 at 5 04 32 PM" src="https://user-images.githubusercontent.com/1689238/120390393-a671fc80-c2fb-11eb-8235-39decb84b022.png"> | <img width="1061" alt="Screen Shot 2021-06-01 at 5 05 11 PM" src="https://user-images.githubusercontent.com/1689238/120390420-af62ce00-c2fb-11eb-862f-2b2131dcf83f.png"> | <img width="1059" alt="Screen Shot 2021-06-01 at 5 05 32 PM" src="https://user-images.githubusercontent.com/1689238/120390441-b4278200-c2fb-11eb-84e9-ecc2e2913b44.png">
<img width="1047" alt="Screen Shot 2021-06-01 at 5 04 41 PM" src="https://user-images.githubusercontent.com/1689238/120390518-cc979c80-c2fb-11eb-865d-bd4e603d19bb.png"> | <img width="1053" alt="Screen Shot 2021-06-01 at 5 05 19 PM" src="https://user-images.githubusercontent.com/1689238/120390526-ce616000-c2fb-11eb-87a8-d6958425ae20.png"> | <img width="1050" alt="Screen Shot 2021-06-01 at 5 05 41 PM" src="https://user-images.githubusercontent.com/1689238/120390531-d0c3ba00-c2fb-11eb-9764-612d604a2e3f.png">


[
Follow-up needed](https://github.com/Automattic/wp-calypso/issues/54555) to fix the upsell nudge link plan slugs for free Atomic (should be business-bundle and value_bundle).

Fixes #51828
